File: pset1/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int height;
    char space = ' ';
    char hash = '#';
    do {
        printf("Height: ");
        height = get_int();
    } while ( height < 0 || height > 23);
    int i;
    int j;
    for (i = 1; i <= height; i++) {
        for (j = 1; j <= height; j++) {
            if ( j <= height - i) {
                printf("%c", space);
            } else {
                printf("%c", hash);
            }
        }
    printf("%c", hash);
    printf("\n");
    }
}

File: pset6/mario/mario.py
from cs50 import get_int

height = -1
space = ' '
block = '#'

while height < 0 or height > 23:
    print('Height: ')
    height = get_int()

for x in range(height):
    for y in range(height):
        if y < height - x - 1:
            print(space, end='')
        else:
            print(block, end='')
    print(block)